### PR TITLE
DSDEGP-1909 Force latest version of Jackson everywhere.

### PR DESCRIPTION
**Pull request by danxmoran**
### Description

Also change the URL in the env integration tests to get them passing.

In my last PR I thought I had pulled us to a good version of Jackson by upgrading elastic4s, but this only affected the `clio-server` and `clio-dataaccess-model` subprojects. The other subprojects still depend on an old Jackson version via the google auth library pulled in by `clio-util`. Forcing the latest version in that subproject should fix it everywhere.

----

### Checklist (never delete this)

Never delete this, it is our record that procedure was followed. If you find that for whatever reason one of the checklist points doesn't apply to your PR, you can leave it unchecked but please add an explanation below.

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: Ensure that you have added or updated tests
- [ ] **Submitter**: If you're adding/switching libraries or otherwise changing the design, update the [design documentation](https://broadinstitute.atlassian.net/wiki/pages/viewpage.action?pageId=114531509).
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
* Review cycle:
  * Team may comment on PR at will
  * **Reviewer assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to reviewer** for further feedback
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Move the JIRA issue to QA once this checklist is completed

### CODE REVIEW

File: project/src/main/scala/org/broadinstitute/clio/sbt/Dependencies.scala [hunk 1]
```diff
@@ -18,7 +18,8 @@ object Dependencies {
   private val EnumeratumCirceVersion = "1.5.15"
   private val FicusVersion = "1.4.3"
   private val GoogleAuthHttpVersion = "0.9.0"
-  private val GoogleCloudNioVersion = "0.32.0-alpha"
+  private val GoogleCloudNioVersion = "0.33.0-alpha"
```
Comment (danxmoran) [inline project/src/main/scala/org/broadinstitute/clio/sbt/Dependencies.scala:21]: New version released overnight.

File: clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/EnvIntegrationSpec.scala [hunk 2]
```diff
@@ -42,8 +44,12 @@ abstract class EnvIntegrationSpec(env: String)
     new GoogleCredentialsGenerator(googleCredential)
   )
 
+  /*
+   * TODO: After the new dev cluster is up and running behind elasticsearch1,
+   * change this URL back so it'll track whatever cluster is in use in the environment.
+   */
   override val elasticsearchUri: Uri = Uri(
-    s"http://elasticsearch1.gotc-$env.broadinstitute.org:9200"
+    s"http://elasticsearch101.gotc-$env.broadinstitute.org:9200"
```
Comment (danxmoran) [inline clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/EnvIntegrationSpec.scala]: This broke the nightly integration tests.

File: clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/EnvIntegrationSpec.scala [hunk 3]
```diff
@@ -32,6 +32,8 @@ abstract class EnvIntegrationSpec(env: String)
     )
 
   override val clioWebClient: ClioWebClient = new ClioWebClient(
+    // TODO: Change this to point at a DNS, so it'll track whatever Clio
```
Comment (pshapiro4broad) [inline clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/EnvIntegrationSpec.scala]: This TODO is unnecessary as there is a JIRA for it, DSDEGP-1975

**Review by pshapiro4broad — APPROVED** [review]
👍 once TODOs are removed

**Pull request merged**